src/builtins/Transformers.jl

```diff
@@ -928,8 +926,10 @@ function MMI.transform(transformer::OneHotEncoder, fitresult, X)
             pairs = d[ftr]
             refs = first.(pairs)
             names = last.(pairs)
-            cols_to_add = map(refs) do ref
-                float.(_hot(col, ref))
+            cols_to_add = map(refs) do ref 
+                if typeof(ref) != Missing float.(hot(col, ref)) 
+                else missing # TODO - Make a Vector that contains length(refs) * missing so that it matches the remaining encoded features.
```

Comment:
Could you elaborate on the "TODO" comment. I don't quite follow. 

Are you referring to new feature to be added in a later PR?

Reply:
Hmm.. that's just a note I wrote while I'm working on this. I forgot to remove that. Sorry!